homeassistant/components/vicare/sensor.py

```diff
@@ -488,6 +489,24 @@ class ViCareSensorEntityDescription(SensorEntityDescription, ViCareRequiredKeysM
         device_class=SensorDeviceClass.TEMPERATURE,
         state_class=SensorStateClass.MEASUREMENT,
     ),
+    ViCareSensorEntityDescription(
+        key="active_program",
+        name="Active program",
+        device_class=SensorDeviceClass.ENUM,
+        options=[
+            HeatingProgram.ACTIVE,
+            HeatingProgram.COMFORT,
+            HeatingProgram.ECO,
+            HeatingProgram.EXTERNAL,
+            HeatingProgram.FIXED,
+            HeatingProgram.FORCED,
+            HeatingProgram.HOLIDAY,
+            HeatingProgram.NORMAL,
+            HeatingProgram.REDUCED,
+            HeatingProgram.STANDBY,
+        ],
```

Comment:
This might also work:

```suggestion
        options=[*HeatingProgram],
```